.editorconfig

```diff
@@ -6,3 +6,19 @@ root = true
 # as some compilers complain about files not ending in newline
 [*]
 insert_final_newline = true
+end_of_line = lf
```

Comment:
I don't think this particular setting would work well for users developing on Windows, did you add it based on personal experience?

Our `$(TOP)/.gitattributes` file currently tells Git to check out most text files using the native EOL for the host OS, and convert them back to line-feeds when committing changes. Since Git has been getting this right for a long time I'm not in favor of changing it. I suspect there are editors on Windows that could get confused, and I'd rather leave this to Git.

Reply:
It is to defend against users developing on Windows mixing in their '\r\n' line terminators in case they mis-configure git. I am currently checking where it breaks and what I have to change to make the current code match. Maybe I switch this PR to a draft for the moment...

Reply:
I have cloned this branch in windows and see what you meant.
I have removed end_of_line from .editorconfig.